sycl/test-e2e/DeviceLib/string_test.cpp

```diff
@@ -1,7 +1,10 @@
 // UNSUPPORTED: hip
 // RUN: %{build} -fno-builtin -o %t.out
 // RUN: %{run} %t.out
-
+// TODO: Remove unsupported after fixing
+// https://github.com/intel/llvm/issues/12683
+// UNSUPPORTED: accelerator
```

Comment:
accelerator or fpga?

Reply:
I think it's accelerator only because that's what the name of the feature is (https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/lit.cfg.py#L678) when fpga device is reported by sycl-ls.